### DISCUSSION TITLE
Mac OS .shepherd-element position issue

### DISCUSSION THREAD

**Discussion by Evirtual** [shipshapecode/shepherd#2377] [General]
Everything works fine on windows. But for Mac OS (tested in all browsers), position of `.shepherd-element` is not with focused elements (it is positioned to left top corner of the browser). Once I resize window it goes to the correct position. By the way, modalOverlayOpening is working perfectly.

**Comment by RobbieTheWagner** [comment]
@Evirtual is your issue resolved?

**Reply by Evirtual** [reply]
Hey @RobbieTheWagner, yes issue is resolved. Thanks. I had to set max-width on `.shepherd-element` to make it work correctly for Mac OS.
